src/pages/excursions.js

```diff
@@ -1,87 +1,65 @@
-import React from 'react';
-import Cabin from '../images/cabin.jpg'; 
+import Cabin from '../images/cabin.jpg';
 import '../App.css';
 
+export default function Excursions() {
+	return (
+		<div className='container'>
 
+			<h1 className='about_h1' style={{
```

Comment:
An `<h1>`-tag named "about_**h1**" might be a bit "No shit, Sherlock". ;)
And why add the className and the do inline styling? Now I am utterly confused. ;p
You use style sheets. So don't do inline styling all of a sudden.